paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -70,6 +81,7 @@ public class AccessibilityRenderExtension : RenderExtension {
     }
   }
 
+  @SuppressLint("VisibleForTests")
```

Comment:
nit: revert

You may recall this is because we should be moving away from [ViewRootForTest](https://github.com/cashapp/paparazzi/pull/1580#discussion_r1759282744)